test/Proxy.specs.ts

```diff
@@ -30,7 +30,7 @@ describe("NEW::: Proxy ", async () => {
       let proxy = await Proxy.deploy(implementationAddress);
       await proxy.deployed();
       proxy = await ethers.getContractAt("SmartAccount", proxy.address);
-      await expect(proxy.getChainId()).to.be.reverted;
+      await expect(proxy.getImplementation()).to.be.reverted;
```

Comment:
what is this test case about?

Reply:
it shows that if a proxy has been deployed with an EOA as implementation (not even through Factory), it won't be callable.